### DISCUSSION TITLE
[Detector Support]: Reolink camera detector

### DISCUSSION THREAD

**Discussion by freemax77** [blakeblackshear/frigate#13014] [Ask A Question] [answered]
### What is your question:

hello everyone
In frigate I use OPENVINO detectors.
My Reolink cameras include motion sensors for people, cars and animals. The detection of reolink is more precise than OPENVINO especially with fast movements
Is it possible to use the cameras as detectors in Frigate?
Many thanks

**Comment by hawkeye217** [comment, marked as answer]
No, it’s not possible for frigate to use a camera’s motion detection or object detection. Even if the camera firmware somehow exposed this information and made it available, it likely wouldn’t provide the kind of information frigate uses internally for tracking objects.

**Reply by freemax77** [reply]
Many thanks for the reply.
I had the idea because in Home Assistant the camera detectors are used and I saw that they work very well, but I really like how Frigate works

Do you have any tips to improve the performance of the OpenVino detector?
Here is the configuration:
detectors:
  ov:
    type: openvino
    device: AUTO
model:
  width: 300
  height: 300
  input_tensor: nhwc
  input_pixel_format: bgr
  path: /openvino-model/ssdlite_mobilenet_v2.xml 
  labelmap_path: /openvino-model/coco_91cl_bkgr.txt

Thanks again

**Reply by hawkeye217** [reply]
Happy to help.

What version of Frigate are you using? Are you seeing slow performance with OpenVINO? What is your inference speed?

**Reply by NickM-27** [reply]
To be clear it is possible with something like home assistant to create manual events any time the camera sensors detect something. But that is a lot of effort for less control of detection

**Reply by freemax77** [reply]
> Happy to help.
> 
> What version of Frigate are you using? Are you seeing slow performance with OpenVINO? What is your inference speed?

: )
The version is: Frigate (0.13.2-6476f8a)
The performances are good
Installed on: Intel i5 7400 + 8gb ram + ssd 250gb + (for storage) hdd 1 tb
The interference speed is 14,37 ms 

Thank you

**Reply by hawkeye217** [reply]
A 14ms inference time is good.

**Reply by freemax77** [reply]
> To be clear it is possible with something like home assistant to create manual events any time the camera sensors detect something. But that is a lot of effort for less control of detection

Hello and thanks for the reply
Currently the cameras are installed in both Fregate and Home Assistant
In home assistant I created 2 automations for each camera, when it detects people, one automation sends a notification to the mobile phone, the other creates a snapshot displayed on the home assistant dashboard.
What I wanted is for Fregate to use the camera detector to create the Persons, Car and Dog events

Thanks

**Reply by NickM-27** [reply]
see https://docs.frigate.video/integrations/api#post-apieventscamera_namelabelcreate

**Reply by freemax77** [reply]
> see https://docs.frigate.video/integrations/api#post-apieventscamera_namelabelcreate

Interesting.
I can try
Thanks a lot
